StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetUpdateContext.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  PaymentSheetUpdateContext.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 3/24/23.
+//
+
+import Foundation
+
+struct UpdateContext {
+    /// The status of the last update API call
+    var latestStatus: Status?
+    /// The ID of the last update API call
+    var latestID: UUID?
```

Comment:
these should both be non-optional.  The FlowController property should be optional.